### ISSUE TITLE
Internal server error in column POST method

### ISSUE THREAD

**Issue by karlcz** [informatics-isi-edu/ermrest#78]
```
  mycurl -H Content-Type: application/json -T /tmp/smoketest-7690-test-data -X POST https://test.host/ermrest/catalog/48/schema/test1/table/test_level2/column
  Expected result: 201::*::*
  Actual result: 500::text/html; charset=iso-8859-1::527

  Request body:
    { "type": { "typename": "text" }, "name": "name" }

  Response headers:
    HTTP/1.1 100 Continue
    
    HTTP/1.1 500 Internal Server Error
    Date: Fri, 27 May 2016 00:25:51 GMT
    Content-Length: 527
    Connection: close
    Content-Type: text/html; charset=iso-8859-1
```

Apache ssl_error_log:

```
...
File "/usr/lib/python2.7/site-packages/ermrest/model/column.py", line 63, in sql_comment_resource
    sql_identifier(unicode(self.table.schema.name)),
AttributeError: 'NoneType' object has no attribute 'schema'
```

**Comment by karlcz** [comment]
This is now fixed in master

**Comment by karlcz** [comment]
@robes see fix in d4eae16e10aad3f92e3354412ed51d6b2281a191  and test cases in 55bf48eefe5094486b64d2c640ed9eaefed1f035

**Comment by robes** [comment]
tested it it.
